file_paths.md

```diff
@@ -163,156 +163,166 @@ and `pxp-agent`, all running as `LocalSystem` by default. It will not
             modules *
-        hiera.yaml *                          # :hiera_config
-        hieradata *                           # n/a
-        modules *                             # user modulepath
+        hiera.yaml                            # :hiera_config
+        modules                               # user modulepath
+
+    C:\ProgramData\PuppetLabs\mcollective *
+        client.cfg *
+        facts.yaml *
+        server.cfg *
 
-    C:\ProgramData\PuppetLabs\facter *                                       same
+    C:\ProgramData\PuppetLabs\puppet          # :confdir
+        auth.conf *                           # :rest_authconfig
+        autosign.conf                         # :autosign
+        binder_config.yaml                    # :binder_config
+        csr_attributes.yaml                   # :csr_attributes
+        custom_trusted_oid_mapping.yaml       # :trusted_oid_mapping_file
+        device.conf                           # :deviceconfig
+        fileserver.conf                       # :fileserverconfig
+        puppet.conf *                         # :config
+        routes.yaml                           # :route_file
+        ssl                                   # :ssldir
+
+    C:\ProgramData\PuppetLabs\pxp-agent
+        modules *                             # stores configuration files for pxp-agent modules
+            pxp-module-puppet.conf *          # configuration file of the pxp module puppet (optionally, to override puppet.bat location)
+        pxp-agent.conf                        # pxp-agent configuration file
+
+    C:\Program Files\Puppet Labs\bin *        # symlink targets of puppet related binaries
+        environment.bat@ *                    -> C:\Program Files\Puppet labs\puppet\bin\environment.bat
+        puppet.bat@ *                         -> C:\Program Files\Puppet labs\puppet\bin\puppet.bat
+        hiera.bat@ *                          -> C:\Program Files\Puppet labs\puppet\bin\hiera.bat
+        mco.bat@ *                            -> C:\Program Files\Puppet labs\puppet\bin\mco.bat
+        facter.bat@ *                         -> C:\Program Files\Puppet labs\puppet\bin\facter.bat
+        puppet_shell.bat@ *                   -> C:\Program Files\Puppet labs\puppet\bin\puppet_shell.bat
+        run_facter_interactive.bat@ *         -> C:\Program Files\Puppet labs\puppet\bin\run_facter_interactive.bat
+        run_puppet_interactive.bat@ *         -> C:\Program Files\Puppet labs\puppet\bin\run_puppet_interactive.bat
+
+    C:\Program Files\Puppet Labs\facter *
         facts.d *                             # external facts directory (not pluginsync'ed)
 
-    C:\ProgramData\PuppetLabs\mcollective
+    C:\Program Files\Puppet Labs\mcollective *
         plugins *                             # user installed plugins
-        etc *                                                                same
-            client.cfg
-            facts.yaml
-            server.cfg
```

Comment:
Note, these were moved from `C:\ProgramData\PuppetLabs\mcollective\etc` to `C:\ProgramData\PuppetLabs\mcollective` and the plugins directory was moved from `C:\ProgramData\PuppetLabs\mcollective\plugins` to `C:\Program Files\Puppet Labs\mcollective\plugins`. If this is wrong, we can change it